File: src/main/resources/static/addbuddy.js
$(document).ready(function() {
    $.ajax({
        url: "localhost:8081/addBuddy"
    }).then(function(data) {
        $('.newbud-id').append(data.id);
        $('.newbud-name').append(data.name);
        $('.newbud-phone').append(data.phone);
    });
});

File: src/main/java/Lab06/Controllers/AddBuddy.java
package Lab06.Controllers;

import Lab06.Models.Book;
import Lab06.Models.Buddy;
import Lab06.Models.Newbud;

import org.springframework.stereotype.Controller;
import org.springframework.ui.Model;
import org.springframework.web.bind.annotation.*;

@RestController
public class AddBuddy {
    private Book book = new Book();

    @GetMapping("/addBuddy")
    public Newbud addBuddy(@RequestParam String name, @RequestParam String phone) {
        return new Newbud(name, phone);
    }

    @PostMapping("/addBuddy")
    public Newbud addBuddySubmit(@RequestParam String name, @RequestParam String phone) {
        return new Newbud(name, phone);
    }

    @GetMapping("/addressBook")
    public String getBook(Model model) {
        book.setString();
        model.addAttribute("book", book);
        return "addressBook";
    }
}
